reana_cluster/configurations/reana-cluster.yaml

```diff
@@ -21,12 +24,13 @@ components:
     environment:
       - <<: *db_base_config
       - SHARED_VOLUME_PATH: "/reana"
-      - REANA_STORAGE_BACKEND: "LOCAL"
+      - REANA_STORAGE_BACKEND: "CEPHFS"
 
   reana-server:
     type: "docker"
     image: "reanahub/reana-server:0.3.1"
     environment:
+      - SHARED_VOLUME_PATH: "/reana"
```

Comment:
What about `/reanadb`? Might be good for people trying REANA on a multi-node setup without wanting to use an external DB...